test/test_cuda.py

```diff
@@ -527,16 +527,6 @@ def test_cat_autogpu(self):
         z = torch.cat([x, y], 0)
         self.assertEqual(z.get_device(), x.get_device())
 
-    def test_serialization(self):
```

Comment:
Happy to rename it instead, but if you look at the two definitions it seems like the first one is what you want. Unfortunately, only the second test is running currently, since it clobbers the first definition.